scss/_mixins.scss

```diff
@@ -15,10 +15,17 @@
 	@return $aggregated-selector;
 }
 
-/// Sets SVG background retrieved from the image service
-@mixin oFtIconsGetSvg($icon-name, $color: null, $size: 20, $icon-only: false, $use-local-assets: true) {
-	$service-url: "https://image.webservices.ft.com/v1/images/raw/fticon:";
-	$icon-url: "#{$icon-name}";
+/// Get an SVG icon with PNG fallback for ie6-8
+///
+/// @param {String} $icon-name - this should be a reference to an icon included in o-ft-icons eg arrow-down
+/// @param {String} $color - this should be a hex colour value. Used to color the icon.
```

Comment:
I'd say _an o-colors function_ as technically, we don't provide variable
for colours, but we have functions that output colours

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*